pyproject.toml

```diff
@@ -2,9 +2,13 @@
 name = "pykalman"
 version = "0.9.7"
 description = "An implementation of the Kalman Filter, Kalman Smoother, and EM algorithm in Python"
+maintainers = [
+    {name = "pykalman developers", email = "pykalman@gmail.com"}
```

Comment:
email address is incorrect - I suggest to put info@sktime.net here, that is a catch-all address